lib/index.js

```diff
@@ -5,8 +6,7 @@ var shouldRunTest = (testName, options) => {
   var exclude = options.exclude || [];
 
   if (options.device == 'mobile') {
-    exclude = new Set(exclude.concat(assertions.mobileExclusions));
-    exclude = [...exclude];
+    exclude = exclude.concat(assertions.mobileExclusions);
```

Comment:
Since this array is only used like `exclude.indexOf(testName) == -1`, I think it is fine with duplicates.